flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1119,6 +1124,7 @@ def test_detach_manifestation(self):
         )
 
 
+# FLOC-1874 - Skip or remove all these tests.
```

Comment:
Comment should be addressed. ELsewhere I suggested deleting this; fixing tests so they pass is the other alternative.